docs/Performance.md

```diff
@@ -69,3 +84,41 @@ Measured memory usage (in MB) for particular Events Per Second (EPS) average mes
 | **1000** | 121.6  | 126.75 | 127.94 | 140.11 | 106.82 |
 | **1500** | 128.54 | 131.9  | 137.69 | 95.21  | 113.89 |
 | **2000** | 130.62 | 125.27 | 144.59 | 98.62  | 134.61 |
+
+## Fine Tuning
+
+There are a couple configuration options that can help with performance in specific scenarios.
+
+### Sumo Logic Exporter
+
+The [sumologicexporter](../pkg/exporter/sumologicexporter)
+sends data to Sumo Logic.
+
+It has the following features that can help with performance:
+
+- `retry_on_failure` with its `initial_interval`, `max_interval` and `max_elapsed_time` settings,
+- `sending_queue` with its `num_consumers`, `queue_size` settings,
+- `timeout`.
+
+Read more about these features in the [Sumo Logic Exporter docs](../pkg/exporter/sumologicexporter/README.md).
+
+### Batch Processor
+
+The [batchprocessor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor)
+joins records of each type in batches.
+
+It has the following features that can help with performance:
+
+- `send_batch_size`,
+- `send_batch_max_size`,
+- `timeout`.
+
+Read more about these features in the [Batch Processor docs](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md).
+
+### Memory Limiter Processor
+
+The [memorylimiterprocessor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor)
+prevents out-of-memory crashes for the collector process
+by monitoring the amount of memory used by the collector and forcing it to lower its memory consumption.
+
+Read more about its features in the [Memory Limiter Processor docs](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md).
```

Comment:
One nitpick from my side would be to extract those long URLs as we did elsewhere e.g. this file in line 13

Reply:
Hm I don't see the point in doing that, it just creates redundancy in my opinion..

Reply:
But OK let's do it and let it sink in for some time ;)